### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by ChrisJohnNOAA**
This splits the documentation into multiple files. Hopefully structuring it better for people to find what they're looking for. For the moved documentation it is now in markdown files.

Adds new information about the project license, project governance, and roadmap.

Adds TM to ERDDAP™️.

### CODE REVIEW

**Comment by mwengren** [comment]
@ChrisJohnNOAA Looks like my [PR #151](https://github.com/ERDDAP/erddap/commit/04061484204eb6b3cedadaf753272f3998f7c2ed) to update the main README a few weeks ago must have inadvertently re-added the /development/README.md file.  

This had been moved to /development/docker/README.md in a previous PR.  The /development/README.md is redundant and now has some incorrect paths in it as well.

Perhaps you could delete the duplicate as part of this PR?  

Otherwise, looks good to me.

File: TECHNICAL_BOARD.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+The Technical Board is responsible for the following documents and processes:
+
+* **Review and Approve new features** that are proposed in GitHub Discussions.
+
+* **ERDDAP™ Priority Projects** list, which determine projects that we are working on. Check out the current [priority projects list](PRIORITY_PROJECTS.md).
+
+* **ERDDAP™ Release Feature Goals**, which determine specific features which we aim to include in each release of ERDDAP™. These are documented in the GitHub project for each release.
+
+* **ERDDAP™ Style Guide** which specifies coding styles and practices to be followed in ERDDAP™ code.The ERDDAP™ Technical Board operates on the following basis:* Where possible communication of the Technical Board should be public and asynchronous. In many cases this means utilizing GitHub Discussions and Issues.
```
Comment (mwengren) [inline TECHNICAL_BOARD.md]: Should the sentence starting with 'The ERDDAP Technical Board operates...' be moved to a separate line or bullet?  Seems unrelated to the Style Guide.

Comment (ChrisJohnNOAA) [reply TECHNICAL_BOARD.md]: Good catch! Done.

**Comment by ChrisJohnNOAA** [comment]
> @ChrisJohnNOAA Looks like my [PR #151](https://github.com/ERDDAP/erddap/commit/04061484204eb6b3cedadaf753272f3998f7c2ed) to update the main README a few weeks ago must have inadvertently re-added the /development/README.md file.
> 
> This had been moved to /development/docker/README.md in a previous PR. The /development/README.md is redundant and now has some incorrect paths in it as well.
> 
> Perhaps you could delete the duplicate as part of this PR?
> 
> Otherwise, looks good to me.

Done
